apps/s_server.c

```diff
@@ -101,26 +107,18 @@ static void free_sessions(void);
 static DH *load_dh_param(const char *dhfile);
 #endif
 
-static void s_server_init(void);
-
 /* static int load_CA(SSL_CTX *ctx, char *file);*/
 
 #undef BUFSIZZ
```

Comment:
okay, this whole BUFSIZZ thing is kinda silly now.  maybe it made some sense while ago.  let's make a single constant used in s_client and s_server. Remove the BUFSIZZ from s_time.  Get rid of "bufsize" from s_server.c